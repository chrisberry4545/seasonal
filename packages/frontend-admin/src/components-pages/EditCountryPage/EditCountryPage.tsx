import React, { FC } from 'react';
import { EditCountryForm } from '../../components-main';

export const EditCountryPage: FC<{}> = () => <EditCountryForm />;
