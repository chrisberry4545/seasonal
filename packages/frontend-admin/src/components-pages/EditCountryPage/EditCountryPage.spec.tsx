import React from 'react';
import { EditCountryPage } from './EditCountryPage';
import { shallow, ShallowWrapper } from 'enzyme';

jest.mock('../../components-main', () => ({
  EditCountryForm: () => 'EditCountryForm'
}));

describe('<EditCountryPage />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <EditCountryPage />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
