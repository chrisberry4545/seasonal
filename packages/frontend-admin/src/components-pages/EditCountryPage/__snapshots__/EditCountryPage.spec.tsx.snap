// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditCountryPage /> renders correctly 1`] = `<EditCountryForm />`;
